index.js

```diff
@@ -519,10 +592,14 @@ var assemble = function () {
  */
 module.exports = function (options) {
 
-	// setup assembly
-	setup(options);
+	try {
+
+		// setup assembly
+		setup(options);
+
+		// assemble
+		assemble();
 
-	// assemble
-	assemble();
+	} catch(e) {}
 
```

Comment:
Might be a good idea to have a generic "Unknown" `handleError()` call here just in case.